packages/core/services/AnnotationService/DatabaseAnnotationService/index.ts

```diff
@@ -140,4 +140,13 @@ export default class DatabaseAnnotationService implements AnnotationService {
             }, [] as string[])
             .filter((annotation) => !annotationSet.has(annotation));
     }
+
+    public async createAnnotation(annotation: Annotation): Promise<any> {
+        const tableName = this.dataSourceNames.sort().join(", ");
```

Comment:
I think there is a constant in here that tracks the separater for data source names? Worth checking might be totally wrong.

Reply:
changed in [b3264d1](https://github.com/AllenInstitute/biofile-finder/pull/406/commits/b3264d1b8ea39f7b15e1cf385b839c9561ee0688), 95% sure it's the right constant